src/test/java/com/hubspot/jinjava/objects/collections/PyMapTest.java

```diff
@@ -0,0 +1,262 @@
+package com.hubspot.jinjava.objects.collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.interpret.IndexOutOfRangeException;
+import com.hubspot.jinjava.interpret.RenderResult;
+import com.hubspot.jinjava.interpret.TemplateError;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PyMapTest {
+  private Jinjava jinjava;
+
+  @Before
+  public void setup() {
+    jinjava = new Jinjava();
+  }
+
+  @Test
+  public void itSupportsAppendOperation() {
+    RenderResult result = jinjava.renderForResult(
+      "{% set mymap = {} %} {% do mymap.put('map', mymap) %} {{ mymap }}",
+      Collections.emptyMap()
+    );
```

Comment:
this doesn't assert anything. What should it do?

Reply:
I was using this test for debugging purposes and forgot to reinstate it to its proper form. Thanks for catching this.